apps/src/accounts/AccountInformation/style.module.scss

```diff
@@ -9,7 +9,7 @@
 
     &.verifiedTeacher {
       font-weight: bold;
-      color: $light_affirmative_500;
+      color: var(--sentiment-success-50);
```

Comment:
@drizco can you take a look at this change and the change on `apps/src/accounts/common/common.styles.module.scss`? 

It should have no UI changes, but these were making drone fail. I think it's because I changed the color stylesheet imports [here](https://github.com/code-dot-org/code-dot-org/blob/9094eb5b7369194f2ffcdb1c7116ba6c7bc318f8/apps/src/componentLibrary/typography/typography.module.scss#L1) and [here](https://github.com/code-dot-org/code-dot-org/blob/9094eb5b7369194f2ffcdb1c7116ba6c7bc318f8/apps/src/componentLibrary/common/styles/mixins.scss#L1).